styles.css

```diff
@@ -0,0 +1,733 @@
+}
+.alert2
+{
+border: solid grey;
+border-radius: 0.7rem;
+font-size: 30px;
+background-color: red;
+width: 60%;
+height: 45px;
+padding: 0.5rem 0rem 0rem 0.5rem;
+color: white;
+position: relative;
+}
+.x-div-alert2
+{
+position: absolute;
+right: 8px;
+top:-2px;
+color: white;
+font-size: 18px;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.alert3
+{
+    border: solid grey;
+border-radius: 0.7rem;
+font-size: 30px;
+background-color: rgb(1, 1, 131);
+width: 60%;
+height: 45px;
+padding: 0.5rem 0rem 0rem 0.5rem;
+color: white;
+position: relative;
+
+}
+.x-div-alert3
+{
+position: absolute;
+right: 8px;
+top:-2px;
+color: white;
+font-size: 18px;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.icon-button
+{
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 80px;
+    color: #333333;
+    background: white;
+   border-radius: 50%;
+   border: none;
+    outline: none;
+    
+}
+.icon-button__badge
+{
+    position: absolute;
+    top: -3px;
+    right:22px;
+    width: 15px;
+    height: 15px;
+    background: red;
+    color: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+}
+.sub-heading
+{
+    font-style: italic;
+    color: var(--primary-color);
+}
+.primary-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: var(--primary-color);
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    font-size: 15px;
+}
+.icon-button-buttons
+{
+    padding:0.5rem;
+    color: white;
+    background-color: rgb(40, 185, 3);
+    border: none;
+    outline: none;
+    font-size: 15px;
+   
+}
+.float-action-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: var(--dark-pink);
+    border-radius: 45%;
+    border: none;
+    outline: none;
+    font-size: 15px;
+}
+.link-button{
+    padding: 0.5rem;
+    color: rgb(3, 163, 184);
+    border-radius: 1rem;
+    background-color: white;
+    border-color: rgb(3, 163, 184);
+    font-size: 15px;
+}
+.secondary-button
+{
+    padding: 0.5rem;
+    color: var(--primary-color);
+    background-color: white;
+    border-radius: 0.5rem;
+     border-color: var(--primary-color);
+     outline:none;
+     font-size: 15px;
+}
+.success-badge-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: green;
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    margin-right: 1rem;
+}
+.danger-badge-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color: red;
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    margin-right: 1rem;
+}
+.warning-batch-button
+{
+    padding: 0.5rem;
+    color: white;
+    background-color:rgb(207, 207, 12);
+    border-radius: 0.5rem;
+    border: none;
+    outline: none;
+    margin-right: 1rem;
+}
```

Comment:
Here you can combine `padding, color, border-radius, border, outline, margin-right` in a separate class and another class having only the `background-color`. This will reduce the CSS and will follow the DRY principle.